.github/workflows/release.yml

```diff
@@ -20,9 +24,16 @@ jobs:
         uses: cachix/install-nix-action@v20
 
       - name: Build
-        run: nix build .#packages.x86_64-linux.default --accept-flake-config
-          
+        run: |
+          if [[ "${{ matrix.os }}" == "macos-latest" ]]; then
+            nix build .#packages.x86_64-darwin.default --accept-flake-config
```

Comment:
Isn't this the default behaviour when you simply do `nix build` ?

Reply:
It is, you're right! Let's simplify this :)